src/app/handlers/new_member_role.handler.ts

```diff
@@ -26,8 +27,8 @@ export class NewMemberRoleHandler implements IHandler {
     }
     member.addRole(<Role>this._roleCache[this._UNACKNOWLEDGED_ROLE]);
 
-    const hasAvatar = Boolean(member.user.avatar);
-    if (hasAvatar) {
+    const accountIsNew = MemberUtils.shouldUnverify(member);
```

Comment:
naming logic between `accountIsNew` and `shouldUnverify` is weird, make them consistent